Automattic/pocket-casts-android
# Allow scroll behind mini player

## Description

This updates the bottom inset for the screens so that their content can scroll under the mini player and their bottom padding respects the height of the mini player.

For Fragments
- [x] podcasts
- [x] upnext
- [x] discover
- [x] podcast, grid and list
- [x] filter episodes
- [x] profile episodes
- [x] region select
- [x] podcast settings
- [x] auto-add to up next
- [x] files  <---- has a floating action button at the bottom, test carefully
- [x] account details
- [x] profile <---- has a refresh button at the bottom, test carefully

For Compose pages
- [x] storage settings
- [x] developer settings
- [x] profile bookmarks
- [x] appearance settings
- [x] beta features
- [x] change email
- [x] stats
- [x] advanced settings
- [x] headphone settings
- [x] privacy
- [x] network status

These do not allow scrolling behind mini player but their bottom padding is adjusted according to mini-player height
- [x] help & support
- [x] preferences fragments
       - notification settings
       - auto archive
       - auto download
       - import & export opml
- [x] change pwd


## Testing Instructions

1. Open the app
2. Play an episode
3. Notice that mini player is shown
4. Navigate to different screens 
5. ✅ Notice that the content can be scrolled properly and it doesn't get cut off behind the mini-player
6. ✅ You can see the content through mini-player margins (except for the ones marked explicitly above)
7. Close the mini-player
8.  ✅ Notice that the content bottom inset is adjusted correctly

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/1405144/8db462d8-95e8-4c98-8c25-33c579ee86c3


## Checklist
 
#### I have tested any UI changes...
